assets/javascript/game.js

```diff
@@ -0,0 +1,169 @@
+var dictionary = [
+    "Archaeopteryx",
+    "Baryonyx",
+    "Apatosaurus",
+    "Ankylosaurus",
+    "Borogovia",
+    "Brachiosaurus",
+    "Ceratosaurus",
+    "Compsognathus",
+    "Corythosaurus",
+    "Carcharodontosaurus",
+    "Coelophysis",
+    "Deinonychus",
+    "Dilophosaurus",
+    "Dimetrodon",
+    "Diplodocus",
+    "Dreadnoughtus",
+    "Ekrixinatosaurus",
+    'Echinodon',
+    "Eolambia",
+    "Equijubus",
+    "Fukuiraptor",
+    "Gallimimus",
+    "Gargoyleosaurus",
+    "Giganotosaurus",
+    "Gigantoraptor",
+    "Gondwanatitan",
+    "Fukuisaurus",
+    "Hadrosaurus",
+    "Haya",
+    "Hexing",
+    "Hippodraco",
+    "Huehuecanauhtlus",
+    "Hypsibema",
+    "Ichthyovenator",
+    "Iguanacolossus",
+    "iguanodon",
+    "Indosuchus",
+    "Juratyrant",
+    "Kol",
+    "Lambeosaurus",
+    "Koreaceratops",
+    "Lophorhothon",
+    "Megalosaurus",
+    "Mei",
+    "Megaraptor",
+    "Micropachycephalosaurus",
+    "Mochlodon",
+    "Minotaurasaurus",
+    "Montanoceratops",
+    "Nyasasaurus",
+    "Nodocephalosaurus",
+    "Nanotyrannus",
+    "Oohkotokia",
+    "Oviraptor",
+    "Ozraptor",
+    "Pachycephalosaurus",
+    "Pachyrhinosaurus",
+    "Parasaurolophus",
+    "Pentaceratops",
+    "Pneumatoraptor",
+    "Proa",
+    "Protoceratops",
+    "Pyroraptor",
+    "Qianzhousaurus",
+    "Raptorex",
+    "Riojasaurus",
+    "Saurophaganax",
+    "Santanaraptor",
+    "Sauroposeidon",
+    "Seismosaurus",
+    "Serendipaceratops",
+    "Skorpiovenator",
+    "Stegosaurus",
+    "Spinosaurus",
+    "Struthiomimus",
+    "Stygimoloch",
+    "Styracosaurus",
+    "Supersaurus",
+    "Titanosaurus",
+    "Titanoceratops",
+    "Torosaurus",
+    "Triceratops",
+    "Tuojiangosaurus",
+    "Troodon",
+    "Tyrannosaurus Rex",
+    "Tyrannotitan",
+    "Utahraptor",
+    "Udanoceratops",
+    "Velociraptor",
+    "Velocisaurus",
+    "Vulcanodon",
+    "Wannanosaurus",
+    "Xenoceratops",
+    "Xiongguanlong",
+    "Yutyrannus",
+    "Yongjinglong",
+    "Zby",
+    "Yizhousaurus",
+    "Zuniceratops",
+    "Quetzalcoatlus",
+    "Onychonycteris",
+    "Phthinosuchus",
+];
+
+var gameOn = false;
+var guesses = 10;
+var wrongLetters = [];
+var correctLetters = [];
+var wins = 0;
+var losses = 0;
+var word = dictionary[Math.floor(Math.random() * dictionary.length)];
+var letterCount = word.length;
+var letter = [];
+
+document.onkeyup = function(event) {
+    if (gameOn = false) {
+        setupGame();
+        gameOn = true;
+    }
+    else {
+        if (event.keyCode  >= 65 && event.keyCode <=90) {
+        letter.push(event.key.toLowerCase());
+        alert(letter[0])
```

Comment:
This only alerts the very first letter you type in and doesn't update. Also try not to use alerts anymore. Use `console.log()` to test and remember to remove them when you deploy your page.